databricks/koalas/indexes.py

```diff
@@ -399,6 +400,140 @@ def copy(self, name=None):
             result.name = name
         return result
 
+    def value_counts(
+        self, normalize=False, sort=False, ascending=False, dropna=True
+    ):
+        """
+        Return a Series containing counts of unique values.
+        The resulting object will be in descending order so that the
+        first element is the most frequently-occurring element.
+        Excludes NA values by default.
+
+        Parameters
+        ----------
+        normalize : boolean, default False
+            If True then the object returned will contain the relative
+            frequencies of the unique values.
+        sort : boolean, default False
```

Comment:
`default True`?

Reply:
@ueshin Thanks for the comment!

I set the default value of `sort` as `False` intentionally for avoid performance decreasing.

but maybe is it the better way to set this `True` in terms of mimic pandas'?